ods_ci/tests/Resources/Common.robot

```diff
@@ -305,3 +305,39 @@ Extract URLs From Text
     [Arguments]    ${text}
     ${urls}=     Get Regexp Matches   ${text}   (?:(?:(?:ftp|http)[s]*:\/\/|www\.)[^\.]+\.[^ \n]+)
     RETURN    ${urls}
+
+Run And Watch Command
+  [Documentation]    Run any shell command (including args) with optional:
+  ...    Timeout in minutes: 10 by default.
+  ...    Expected text: Verify the text exists in command output.
+  [Arguments]    ${command}    ${timeout_min}=10    ${expected_text}=${NONE}
```

Comment:
Added your suggestions, thanks!

Reply:
Tested again on https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1849/console

```
Can Install RHODS Operator                                            Cluster type: selfmanaged
Installing RHODS operator in selfmanaged
Watching command output: cd /home/jenkins/workspace/rhods-ci-pr-test/1849/ods-ci/ods_ci/rhodsolm && ./setup.sh -t operator -u beta -i brew.registry.redhat.io/rh-osbs/iib:470465
Shell process started in the background
..namespace/redhat-ods-operator created
Context "default/api-ods-qe-07-rhods-ccitredhat-com:6443/htpasswd-cluster-admin-user" modified.
catalogsource.operators.coreos.com/rhods-catalog-dev created
operatorgroup.operators.coreos.com/rhods-operator-dev created
subscription.operators.coreos.com/rhods-operator-dev created
```

@jgarciao , @bdattoma  can you review to merge ?

Reply:
the new changes looks good to me. Only one thing I believe it's better to change, sorry I didn't notice before. Could you replace all the calls to `Run Keyword If` with an `IF` statement? that kw is deprecated and going to be removed in next RF upgrade

Reply:
@manosnoam , in https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1850/console I see this error:
```
Multiple keywords with name 'Get RHODS Version' found. Give the full name of the keyword you want to use:
    InstalledOperators.Get RHODS version
    oc_version.Get RHODS Version
```

I believe it's not introduced in this PR but warning you just in case